contributing/code/patches.rst

```diff
@@ -113,7 +113,9 @@ Choose the right Branch
 Before working on a patch, you must determine on which branch you need to
 work:
 
-* ``2.3``, if you are fixing a bug for an existing feature or want to make a
+* ``2.3``, for security issues;
```

Comment:
Yes, let's remove this about security fixes. We already have some warnings explaining how to deal with security issues.